docs/index.md

```diff
@@ -16,12 +16,16 @@ For more of the rationale behind Autolab, please check out <a href="https://auto
  -->
 ##Components
 
-Autolab consists of two services: (1) the Ruby on Rails frontend, and (2) [Tango](/tango), the RESTful Python autograding server. <b>Either service can run independently without the other</b>. But in order to use all features of Autolab, we highly recommend installing both services.
+Autolab consists of two services: (1) the Autolab frontend which is implemented using Ruby on Rails, and (2) Tango, the RESTful Python autograding server. <b>Either service can run independently without the other.</b> But in order to use all features of Autolab, we highly recommend installing both services.
```

Comment:
The link to tango is removed since based on our discussion last time we will get rid of the tango basics (which tango.md points to) altogether.